staging/src/k8s.io/apimachinery/pkg/runtime/conversion.go

```diff
@@ -82,32 +84,91 @@ func Convert_Slice_string_To_int(input *[]string, out *int, s conversion.Scope)
 	return nil
 }
 
+func string_to_bool(in string) bool {
+	switch strings.ToLower(in) {
```

Comment:
suggest using strings.EqualFold:

```go
switch {
  case in == "0", strings.EqualFold(in, "false"):
    return false

  default:
    return true
}
```

Reply:
include the godoc about what values mean what... zero-length meaning true is not intuitive

Reply:
done